CSharp/DevVmPowershell/Helpers/EnvironmentVariableHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Helpers
+{
+	public class EnvironmentVariableHelper : IEnvironmentVariableHelper
+	{
+		public void UpdateJavaEnvironmentVariables()
+		{
+			try
+			{
+				// Get the full Java Path
+				string[] directories = Directory.GetDirectories(Constants.EnvironmentVariables.JavaPath);
+				if (directories.Length == 0)
+				{
+					throw new Exception("Java Path is invalid");
+				}
+				string fullJavaPath = directories.First();
+
+				// Update Java Environment Variables
+				Environment.SetEnvironmentVariable(Constants.EnvironmentVariables.KcuraJavaHome, fullJavaPath, EnvironmentVariableTarget.Machine);
+				Environment.SetEnvironmentVariable(Constants.EnvironmentVariables.JavaHome, fullJavaPath, EnvironmentVariableTarget.Machine);
+				Console.WriteLine("Successfully Updated the Java Environment Variables");
```

Comment:
Can we query for the environment variable after the update to be sure that it's actually updated?